src/main/java/net/sf/jabref/model/database/BibDatabase.java

```diff
@@ -507,7 +527,7 @@ public static String getResolvedField(String field, BibEntry entry, BibDatabase
             return EntryUtil.capitalizeFirst(entry.getType());
         }
 
-        // TODO: Changed this to also consider alias fields, which is the expected
+        // Changed this to also consider alias fields, which is the expected
         // behavior for the preview layout and for the check whatever all fields are present.
         // But there might be unwanted side-effects?!
```

Comment:
If you remove the `TODO:` label there shouldn't be an unanswered question in the comments anymore.